package.json

```diff
@@ -65,12 +65,13 @@
     "clamp": "^1.0.1",
     "fbjs": "^0.8.5",
     "path-to-regexp": "^1.7.0",
-    "react-addons-shallow-compare": "15.4.1",
```

Comment:
This was a peer dependency in react-native-tab-view sometime ago. It's not needed anymore for a few releases.